code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -0,0 +1,764 @@
+				H.emp_act(1)
+		if("Disable Cameras")
+			for(var/obj/machinery/camera/C in cameranet.cameras)
+				C.emp_act(1)
+			for(var/mob/living/silicon/ai/A in living_mob_list)
+				A << "<span class='userdanger'>Massive energy surge detected. All cameras offline.</span>"
+				A << 'sound/machines/warning-buzzer.ogg'
+		if("Disable Cyborgs")
+			for(var/mob/living/silicon/robot/R in living_mob_list) //Doesn't include AIs, for obvious reasons
+				if(is_servant_of_ratvar(R) || R.stat) //Doesn't affect already-offline cyborgs
+					continue
+				R.visible_message("<span class='warning'>[R] shuts down with no warning!</span>", \
+				"<span class='userdanger'>Massive emergy surge detected. All systems offline. Initiating reboot sequence..</span>")
+				playsound(R, 'sound/machines/warning-buzzer.ogg', 50, 1)
+				R.Weaken(30)
+	user.visible_message("<span class='warning'>The tendrils around [user]'s arm turn to an onyx black and wither away!</span>", \
+	"<span class='heavy_brass'>The tendrils around your arm turn a horrible black and sting your skin before they shrivel away.</span>")
+	user.notransform = FALSE
+	if(!src)
+		return 0
+	flick("[initial(icon_state)]_discharged", src)
+	icon_state = "[initial(icon_state)]_recharging"
+	spawn(3000) //5 minutes
+		if(!src)
+			return 0
+		visible_message("<span class='warning'>The writhing tendrils return to the gemstone, which begins to glow with power.</span>")
+		flick("[initial(icon_state)]_recharged", src)
+		icon_state = initial(icon_state)
+		recharging = FALSE
+	return 1
+
+/obj/structure/clockwork/mending_motor //Mending motor: A prism that consumes replicant alloy to repair nearby mechanical servants at a quick rate.
+	name = "mending motor"
+	desc = "A dark onyx prism, held in midair by spiraling tendrils of stone."
+	clockwork_desc = "A powerful prism that rapidly repairs nearby mechanical servants and clockwork structures."
+	icon_state = "mending_motor"
+	construction_value = 20
+	break_message = "<span class='warning'>The prism collapses with a heavy thud!</span>"
+	var/stored_alloy = 0
+	var/max_alloy = 150
+	var/uses_alloy = TRUE
+	var/active = TRUE
+
+/obj/structure/clockwork/mending_motor/prefilled
+	stored_alloy = 30
+
+/obj/structure/clockwork/mending_motor/New()
+	..()
+	SSobj.processing += src
+	toggle() //Toggles off as soon as it's created, but starts online for reasons
+
+/obj/structure/clockwork/mending_motor/Destroy()
+	SSobj.processing -= src
+	..()
+
+/obj/structure/clockwork/mending_motor/examine(mob/user)
+	..()
+	if(is_servant_of_ratvar(user) || isobserver(user))
+		user << "It has [stored_alloy]/[max_alloy] units of replicant alloy."
+
+/obj/structure/clockwork/mending_motor/process()
+	if(!active)
+		return 0
+	if(!stored_alloy && uses_alloy)
+		visible_message("<span class='warning'>[src] emits an airy chuckling sound and falls dark!</span>")
+		toggle()
+		return 0
+	for(var/atom/movable/M in range(5, src))
+		switch(M.type)
```

Comment:
Bad! You need to do istype(thing, type) or you won't get subtypes.
Same for the clockwork proselytizer, now that I check the code.